tests/integration/conftest.py

```diff
@@ -55,20 +56,35 @@ def create_segy_mock_4d(
         index_receivers = False
 
     shot_headers = np.hstack([np.repeat(shot, total_chan) for shot in shots])
+
+    # calculate gun
+    if guns is None:
+        guns = [
```

Comment:
this case is showing not covered in coverage report. If this test case is invalid, we can remove this, or if its missing it we should add it.